homu/server.py

```diff
@@ -29,6 +29,10 @@ def index():
 
 @get('/queue/<repo_label>')
 def queue(repo_label):
+    logger = g.logger.getChild('queue')
```

Comment:
Hmm, I'm not familiar with `.getChild()`, but isn't it inefficient as the `logger` object will be created every time the queue is accessed? It that a common idiom when using the `logging` module?